### ISSUE TITLE
Intermittently not working on DIV

### ISSUE THREAD

**Issue by Blastnsmash** [zengabor/zenscroll#28]
I plugged this into a project and on the desktop it seemed to work perfectly. But when you shrink the size of the browser window to a certain height I could get the code to periodically not fire (seemingly random)

Changing this line of code from:

```
if (p < 1 && (getViewHeight() + y) < (scrollContainer || docElem).scrollHeight) {
						    loopScroll();
```

to
```

if (p < 1 && (getViewHeight() + y) <=  (scrollContainer || docElem).scrollHeight) {
						    loopScroll();
```
(just added the <= instead of <)
This corrected the issue but I'm not sure if that is going to create infinite loops or other problems.

**Comment by zengabor** [comment]
Thank you, I will look into this. Can you maybe share the code so I can produce it?

**Comment by KyleLWatson** [comment]
http://63.229.69.43:999/MegaNavBar/Kyle2.html

Reproduce on desktop Chrome: 
-Shrink browser size to 320px x 414px
-Click on Golf then Scroll down and click on lodging. Repeat until it misfires. 
-Also reproducible on chrome for android

Please let me know when you're done and I'll take down the link.

**Comment by zengabor** [comment]
What do you mean by Golf? Where is "lodging"?

**Comment by KyleLWatson** [comment]
Did you expand the menu?

**Comment by KyleLWatson** [comment]
Use a browser inspector if you can't find the elements. Let me know.

**Comment by Blastnsmash** [comment]
https://drive.google.com/file/d/0B-M6Yes83t08ZDZMWFBlcmNBOWc/view?usp=sharing

Please see the screen cast. When I wiggle my mouse cursor then it should have scrolled but didn't.

**Comment by zengabor** [comment]
Would you mind testing the 4.0 release candidate?

You can find it here: https://github.com/zengabor/zenscroll/tree/gh-pages

**Comment by Blastnsmash** [comment]
After more testing, I think the issue is that there is some other javaccript that is changing the height of the parent scroller after zen scroll executes. The problem is, I need some javascript to listen to the attribute change event on an element in order to execute zen scroll AFTER the parent div height changed.

For whatever reason making the code change I suggested does resolve the issue. Closing for now.
